lib/CGameInfoCallback.h

```diff
@@ -91,6 +91,7 @@ class DLL_LINKAGE CGameInfoCallback : public virtual CCallbackBase
 	const CMapHeader * getMapHeader()const;
 	int3 getMapSize() const; //returns size of map - z is 1 for one - level map and 2 for two level map
 	const TerrainTile * getTile(int3 tile, bool verbose = true) const;
+	shared_ptr<boost::multi_array<TerrainTile*, 3>> CGameInfoCallback::getAllVisibleTiles() const;
```

Comment:
Shall be shared_ptr<boost::multi_array<TerrainTile*, 3>> getAllVisibleTiles() const;